pkg/apis/mysql/v1alpha1/mysqlcluster_status.go

```diff
@@ -0,0 +1,215 @@
+/*
+Copyright 2018 Pressinfra SRL
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"time"
+
+	// TODO: dont use glog use logutil
+	"github.com/golang/glog"
+
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// UpdateStatusCondition sets the condition to a status.
+// for example Ready condition to True, or False
+func (c *MysqlCluster) UpdateStatusCondition(condType ClusterConditionType,
+	status core.ConditionStatus, reason, msg string) {
+	newCondition := ClusterCondition{
+		Type:    condType,
+		Status:  status,
+		Reason:  reason,
+		Message: msg,
+	}
+
+	t := time.Now()
+
+	if len(c.Status.Conditions) == 0 {
+		glog.V(4).Infof("Setting lastTransitionTime for mysql cluster "+
```

Comment:
Remove glog from `pkg/apis` so that this package can be safely imported and used into other projects.

Reply:
Ok, I moved this logic from `vialpaha1.MysqlCluster` to `pkg/wrappers/mysqlcluster`. In the wrapper package it creates  a new object that contains a `v1alpha1.MysqlCluster` and all helper functions. I did this to separate the data that is defined in `v1alpha1` and needed logic on those objects.